docs/install/remote.md

```diff
@@ -50,67 +50,47 @@ Before you begin, ensure that the following prerequisites are met:
+You have few options to configure your remote environment:
+1. Using the default `.env` file (**Recommended**)
+2. Using your own environment file
+3. Using MLRun SDK or CLI
+4. Using your IDE (e.g PyCharm or VSCode)
 
-### Set environment variables
+### Using the default `.env` file (**Recommended**)
 
-Set environment variables to define your MLRun configuration. As a minimum requirement:
+The default `.env` file is located by default at `~/.mlrun.env` for Linux and `%USERPROFILE%/.mlrun.env` for Windows. This file can be edited manually or with `mlrun config set` command line:
 
-1. Set `MLRUN_DBPATH` to the URL of the remote MLRun database/API service:
-
-    ```ini
-    MLRUN_DBPATH=<URL endpoint of the MLRun APIs service endpoint; e.g., "https://mlrun-api.default-tenant.app.mycluster.iguazio.com">
-    ```
-
-2. If the remote service is on an instance of the Iguazio MLOps Platform (**not relevant for MLRun CE**), set the following environment variables as well:
-
-    ```ini
-    V3IO_USERNAME=<username of a platform user with access to the MLRun service>
-    V3IO_ACCESS_KEY=<platform access key>
-    ```
-
-    You can get the platform access key from the platform dashboard: select the user-profile picture or icon from the top right corner of 
-    any page, and select **Access Keys** from the menu. In the **Access Keys** window, either copy an existing access key or create a new 
-    key and copy it. Alternatively, you can get the access key by checking the value of the `V3IO_ACCESS_KEY` environment variable in a web-shell or Jupyter Notebook service.
-
-
-You can also set the environment using MLRun SDK, for example:
-
-```python
-# Use local service
-mlrun.set_environment("http://localhost:8080", artifact_path="./")
-# Use remote service
-mlrun.set_environment("<remote-service-url>", access_key="xyz", username="joe")
-```
-
-3. Run `mlrun config set` command line to set configuration parameters in mlrun default or specified .env file. By default, it stores all of the configuration into the default environment file, and your own environment file does not need editing.
+Run the `mlrun config set` command line to set configuration parameters in mlrun default or the specified `.env` file. By default, it stores all of the configuration into the default environment file, and your own environment file does not need editing.
 
 The `set` command can work with the following parameters:
-    `--env-file` or `-f` to set the url path to the mlrun .env file (defaults to '~/.mlrun.env' for Linux and to %USERPROFILE%/.mlrun.env for Windows)
-    `--api` or `-a` to set the url (local or remote) for MLRun API
-    `--artifact-path` or `-p` to set the artifact path (optional)
-    `--username` or `-u` to set the username (for remote access)
-    `--access-key` or `-k` to set the access key (for remote access)
-    `--env-vars` or `-e` to set the environment variables (optional)
+    -`--env-file` or `-f` to set the url path to the mlrun `.env file` 
```

Comment:
-`--env-file` or `-f` to set the url path to the mlrun `.env file` 
-`--api` or `-a` to set the url (local or remote) for MLRun API
-`--artifact-path` or `-p` to set the artifact path (optional)
-`--username` or `-u` to set the username (for remote access)
-`--access-key` or `-k` to set the access key (for remote access)
-`--env-vars` or `-e` to set the environment variables (optional)